report/how_to.md

```diff
@@ -0,0 +1,70 @@
+# User Guide
+
+## How to use this book
+
+### Validation
+
+TODO
+
+### Verification
+
+Each verification case was created using either the [`MES`](./verification/methods.md/#method-of-exact-solutions-mes) or [`MMS`](./verification/methods.md/#method-of-manufactured-solutions-mms) and are tagged accordingly.
+
+TODO
+
+### Navigating tags
+
+Each case is tagged and tag's cases can be navigated through the [🏷 Tags](./_tags/tagsindex.md) index.
+
+* [1D](./_tags/1d.md) : One-dimensional simulations.
+* [2D](./_tags/2d.md) : Two-dimensional simulations.
+* [`MES`](./_tags/mes.md) : Cases using the [`MES`](./verification/methods.md/#method-of-exact-solutions-mes)
+* [`MMS`](./_tags/mms.md) : Cases using the [`MMS`](./verification/methods.md/#method-of-manufactured-solutions-mms).
+* ...
```

Comment:
Don't know if we want to detail all these tags it may be annoying to maintain if they change for some reason